bundles/org.openhab.core.io.monitor/pom.xml

```diff
@@ -31,6 +31,24 @@
       <version>1.6.3</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hdrhistogram</groupId>
+      <artifactId>HdrHistogram</artifactId>
+      <version>2.1.12</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.latencyutils</groupId>
+      <artifactId>LatencyUtils</artifactId>
+      <version>2.0.3</version>
+      <scope>runtime</scope>
```

Comment:
Don't you need to add this to the Karaf feature to make the bundle is installed at runtime?